digest.gemspec

```diff
@@ -12,47 +12,28 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/ruby/digest"
   spec.licenses      = ["Ruby", "BSD-2-Clause"]
 
-  spec.files         = [
-    "LICENSE.txt", "README.md",
-
-    "ext/digest/defs.h", "ext/digest/digest.c", "ext/digest/digest.h",
-    "ext/digest/digest_conf.rb", "ext/digest/extconf.rb",
-    "ext/digest/lib/digest.rb",
-
-    "ext/digest/bubblebabble/bubblebabble.c",
-    "ext/digest/bubblebabble/extconf.rb",
-
-    "ext/digest/md5/extconf.rb", "ext/digest/md5/md5.c",
-    "ext/digest/md5/md5.h", "ext/digest/md5/md5cc.h",
-    "ext/digest/md5/md5init.c",
-
-    "ext/digest/rmd160/extconf.rb", "ext/digest/rmd160/rmd160.c",
-    "ext/digest/rmd160/rmd160.h", "ext/digest/rmd160/rmd160init.c",
-
-    "ext/digest/sha1/extconf.rb", "ext/digest/sha1/sha1.c",
-    "ext/digest/sha1/sha1.h", "ext/digest/sha1/sha1cc.h",
-    "ext/digest/sha1/sha1init.c",
-
-    "ext/digest/sha2/extconf.rb", "ext/digest/sha2/lib/sha2.rb",
-    "ext/digest/sha2/sha2.c", "ext/digest/sha2/sha2.h",
-    "ext/digest/sha2/sha2cc.h", "ext/digest/sha2/sha2init.c",
-
-    "ext/openssl/deprecation.rb",
-    "ext/digest/test.sh",
+  spec.files = [
+    "LICENSE.txt",
+    "README.md",
+    *Dir["lib/**/*.rb"],
```

Comment:
Globbing all existing files might cause problems at merging to ruby/ruby.

Reply:
Fixed.